t/plugin/jwt-auth.t

```diff
@@ -1001,25 +1001,20 @@ base64_secret required but the secret is not in base64 format
 --- config
     location /t {
         content_by_lua_block {
-            local t = require("lib.test_admin").test
-            local code, body, res = t('/apisix/admin/consumers',
-                ngx.HTTP_PUT,
-                [[{
-                    "username": "kerouac",
-                    "plugins": {
-                        "jwt-auth": {
-                            "key": "exp-not-set",
-                            "secret": "my-secret-key"
-                        }
-                    }
-                }]]
-            )
-
-            res = require("toolkit.json").decode(res)
-            assert(res.value.plugins["jwt-auth"].exp == 86400)
+            local core = require("apisix.core")
```

Comment:
Because the original method can no longer detect whether the `exp` variable exists, it's changed to direct validation through the schema, according to the internal logic.